project-words.txt

```diff
@@ -1,2 +1,3 @@
 NestJS
 Embeddables
+indeterminated
```

Comment:
What is this? 😅

Reply:
This is for the spellcheck, the word is not in the dictionary, so we must add it in the project-words.txt: https://github.com/vivid-planet/comet/pull/2838

Reply:
What spellcheck? Is this really needed for anything? 😅

Reply:
Actually, now that I think about it, `indeterminated` isn't a word, the correct word would be `indeterminat`. 

Could you please update the icons name to fix that?
Then it matches MUIs prop name and should be valid in any spellcheck (why ever that is relevant 🤷🏼‍♂️)

Reply:
I've changed the name to `CheckboxIndeterminate`, also see docs from MUI https://v5.mui.com/material-ui/api/checkbox/#props